tests/functional/c/comparison_of_constants.txt

```diff
@@ -1,4 +1,4 @@
-comparison-of-constants:3:6:3:12::"Comparison between constants: '2 == 2' has a constant value":HIGH
-comparison-of-constants:6:6:6:11::"Comparison between constants: '2 > 2' has a constant value":HIGH
-comparison-of-constants:16:3:16:15::"Comparison between constants: 'True == True' has a constant value":HIGH
+comparison-of-constants:3:6:3:12::"Comparison between constants: ""2 == 2"" has a constant value":HIGH
```

Comment:
It appears as double ``""`` but it's due to the "csv" encoding (the string start with ``"``) and it's actually fine.